.github/workflows/release-new-version.yml

```diff
@@ -0,0 +1,52 @@
+name: Release a new version
+
+on:
+  schedule:
+    - cron: "30 12 * * *" # Run workflow at 6 PM IST
```

Comment:
@shakthi-22 @SanchithHegde @inventvenkat - should we do this twice a day to accommodate for commits that come in late in the night and in the first half of the day?

Reply:
as discussed offline.  Let us start with one.